src/pages/ReimbursementAccount/BankAccountPlaidStep.js

```diff
@@ -39,103 +39,96 @@ const defaultProps = {
     plaidLinkOAuthToken: '',
 };
 
-class BankAccountPlaidStep extends React.Component {
-    constructor(props) {
-        super(props);
-        this.validate = this.validate.bind(this);
-        this.submit = this.submit.bind(this);
-    }
-
-    validate(values) {
+function BankAccountPlaidStep({plaidData, receivedRedirectURI, plaidLinkOAuthToken, ...props}) {
```

Comment:
We don't recommend destructing props, this is something that is the first point in the [thread](https://expensify.slack.com/archives/C01GTK53T8Q/p1686062674061049?thread_ts=1685727880.152119&cid=C01GTK53T8Q) linked.

The same is part of the PR Author checklist also

- [ ] I verified the JSDocs style guidelines (in [STYLE.md](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed

Quick link for the [destructuring](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring)

Reply:
@Santhosh-Sellavel yup, you're absolutely correct. 

Very recently, I found new updates on this rule and new guideline got changed to use props de-structuring.

Here is the update - https://expensify.slack.com/archives/C01GTK53T8Q/p1687566918003889?thread_ts=1685727880.152119&cid=C01GTK53T8Q

Also, I'm noticing new contributions in migration have started following this guideline update.

Let me know your thoughts if I should update this.

Reply:
If then we move the destructure in a new line, not in the function argument!

`const { var1, var2, etc } = props`

Reply:
updated